lib/graphql/language/nodes.rb

```diff
@@ -270,15 +270,17 @@ def scalars
             "col: nil",
             "pos: nil",
             "filename: nil",
-            "source: nil",
+            "source: nil"
           ]
 
+          IGNORED_MARSHALLING_KEYWORDS = [:comment]
```

Comment:
Why not marshal with the comment? (I'd expect a marshal-unmarshal document to be identical.)

Reply:
The C parser doesn't know about it and complains we require an argument it doesn't have, but we need the comment to exist on the nodes, cause the nodes are used when dumping the schema from the definition. So I made it a sort of default argument.

Maybe there's a better way.

Reply:
:+1: That makes sense, thanks for the context!